server/routes/add-appointment/addAppointmentController.ts

```diff
@@ -1,13 +1,99 @@
 import { RequestHandler } from 'express'
 import RpService from '../../services/rpService'
 import AddAppointmentView from './addAppointmentView'
+import { AppointmentErrorMessage } from '../../data/model/appointmentErrorMessage'
+import { RPClient } from '../../data'
+import logger from '../../../logger'
 
 export default class AddAppointmentController {
   constructor(private readonly rpService: RpService) {}
 
   getAddAppointmentView: RequestHandler = async (req, res, next): Promise<void> => {
     const { prisonerData } = req
+    const params = req.query
     const view = new AddAppointmentView(prisonerData)
-    res.render('pages/add-appointment', { ...view.renderArgs })
+    res.render('pages/add-appointment', { ...view.renderArgs, params })
```

Comment:
Might be better to explicitly put the params we need into the renderArgs rather than just passing it all through

Reply:
All the params are needed for pre-populating the input fields